api/schemas/report.json

```diff
@@ -136,9 +136,8 @@
         },
 
         "kind": {
```

Comment:
I was wondering about that. Happy to do it if you think it won't break too much.